helix-term/src/ui/editor.rs

```diff
@@ -441,6 +441,11 @@ impl EditorView {
             // extra loops if the code is deeply nested.
 
             for i in starting_indent..(indent_level / tab_width as u16) {
+                if config.indent_guides.rainbow {
+                    let color_index: usize = i as usize % theme.rainbow_length();
+                    indent_guide_style.fg = theme.get(&format!("rainbow.{}", color_index)).fg;
```

Comment:
Instead of patching only the foreground, we most likely want to patch the entire style here. We shouldn't assume that indent guides are only ever themed using foreground

Reply:
> Instead of patching only the foreground, we most likely want to patch the entire style here. We shouldn't assume that indent guides are only ever themed using foreground

seems reasonable, i thought about it myself and figured that there may not be a real usecase other than patching the fg.